add-review.txt

```diff
@@ -0,0 +1,51 @@
+-- ds.page: Service API call using `http` processor

```

Comment:
@AviralVerma13 why extension is `.txt`?

Reply:
that was just a scratch file storing testing movie-review page ui.